### ISSUE TITLE
Use local solc

### ISSUE THREAD

**Issue by SCBuergel** [juanfranblanco/vscode-solidity#23]
I have installed solc 0.4.10. However, it seems that the extension ships with some default version that currently is at 0.4.9 (`pragma solidity ^0.4.10;` gives an error). I can change this version to a remote compiler version via the settings. This change requires me to be online. For train-coders like myself it would be very useful to simply use the locally installed solc in whatever version that is.

**Comment by juanfranblanco** [comment]
@SCBuergel  Updated now

**Comment by SCBuergel** [comment]
@juanfranblanco this issue is not (just) about updating the default compiler from 0.4.9 to 0.4.10. It is about using the local solc instead of the default provided one. Is that on the roadmap? Would be fantastic!

**Comment by juanfranblanco** [comment]
@SCBuergel yes it might be possible "downloading" solcjs from local storage.

**Comment by juanfranblanco** [comment]
Note: This might possible using require-from-string and the wrapper setupMethods()

**Comment by SCBuergel** [comment]
@juanfranblanco if you had any more instructions on how to "download" solcjs from local storage that would be helpful. I see that the current version is still on 0.4.10. I just tried simply installing `npm install -g solc` which did not do the trick, the extension seems to always use its own.

**Comment by juanfranblanco** [comment]
@SCBuergel check the latest, now you can use any of the sol-bin files

**Comment by juanfranblanco** [comment]
@SCBuergel  I will be adding a capability (working on it) to install in the project solc from npm and use that as the first option.

1st local 
2nd check settings for local bin file (soljs bin)
3rd check settings for remote version ('latest')
4th finally use the version within vs code
